scripts/inner/kubestellar

```diff
@@ -340,7 +349,7 @@ fi
 
 # Start the Placement Translator
 sleep 3
-placement-translator --allclusters-context  "system:admin" -v=2 >> $log_folder/placement-translator-log.txt 2>&1 &
+placement-translator -v=2 >> $log_folder/placement-translator-log.txt 2>&1 &
```

Comment:
This problem was not introduced in this PR, but there is a disconnect somewhere from the verbosity setting in the Helm chart.